tippicserver/models/transaction.py

```diff
@@ -58,18 +58,38 @@ def report_transaction(tx_json):
 
 
 def get_transactions_json(user_id, public_address, discovery_apps):
-    from tippicserver.utils import MAX_TXS_PER_USER,APP_TO_APP, PICTURE, GIFT, GIVE_TIP, GET_TIP
-    import arrow
+    import arrow, re
+    from tippicserver.utils import MAX_TXS_PER_USER,APP_TO_APP, PICTURE, GIFT, GIVE_TIP, GET_TIP, IN, OUT
     from tippicserver.models.user import get_user
+
     
     detailed_txs = []
     user = get_user(user_id)
+    p = re.compile("1-(.*?)-CrossApps_(.*)")
```

Comment:
you usually put r before a regex pattern i.e r"1-(.*?)-CrossApps_(.*)" to make it a raw string (disregard escape chars)